src/forms/stock-dispense/useDispenseStock.tsx

```diff
@@ -0,0 +1,79 @@
+import useSWR from 'swr';
```

Comment:
I know we've started a pattern of calling something "useDispenseStock" but then including multiple different hooks in it, but I think it would be better to get away from this and call this something like stock.resource.tsx or stock-management.resource.tsx.   Do you agree @denniskigen  @ibacher ?

Reply:
Indeed, that's substantially more consistent with every other app in the ecosystem.